Purpose: Try to get udev, kdm and gdm to start earlier during boot.
Fixes:   None.
Authour: Petter Reinholdtsen
Status:  Not applied upstream
Index: a/makeboot.c
===================================================================
--- a/makeboot.c	(revision 1780)
+++ b/makeboot.c	(working copy)
@@ -236,7 +236,11 @@
 	for (node = tree_list; node; node = node->next) {
 		int importance = 0;
 
-		if (! strcmp(node->name, "xdm"))
+		if (! strcmp(node->name, "xdm")
+                    || ! strcmp(node->name, "udev")
+                    || ! strcmp(node->name, "gdm")
+                    || ! strcmp(node->name, "gdm3")
+                    || ! strcmp(node->name, "kdm"))
 			importance = 100;
 
 		if (! strncmp(node->name, "early", 5))
